src/RTTI/RTTIPlainTypes.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+#include <BsPrerequisites.h>
+
+#include <scripting/ScriptTypes.hpp>
+#include "RTTI_TypeIDs.hpp"
+
+namespace bs
+{
+  /**
+   * RTTIPlainType for std::pair.
+   *
+   * @see		RTTIPlainType
+   */
+  #if 0
```

Comment:
Since it is always "ifdeffed" out, do we need this code/this file?

Reply:
This was me fighting with serializing complex `bs::Map` types. I will remove it.

Reply:
Doesn't it work out of the box, somehow?

Reply:
It works for plain types, such as `bs::Map<int, int>` or even `bs::Map<bs::String, int>`. It even does containers like `bs::map<bs::String, bs::Vector<int>>`. But you're out of luck if you're mapping a generic `IReflectable` such as `ScriptObject`.

Reply:
Removed.